sdk/test-utils/perfstress/src/program.ts

```diff
@@ -82,16 +92,22 @@ export class PerfStressProgram {
    * @param parallels Parallel executions
    */
   private logResults(parallels: PerfStressParallel[]): void {
-    const totalOperations = parallels.reduce((sum, i) => sum + i.completedOperations, 0);
-    const operationsPerSecond = parallels.reduce((sum, parallel) => {
-      return sum + parallel.completedOperations / (parallel.lastMillisecondsElapsed / 1000);
-    }, 0);
+    const totalOperations = this.getCompletedOperations(parallels);
+    const operationsPerSecond = this.getOperationsPerSecond(parallels);
     const secondsPerOperation = 1 / operationsPerSecond;
     const weightedAverage = totalOperations / operationsPerSecond;
     console.log(
-      `Completed ${totalOperations} operations in a weighted-average of ${weightedAverage.toFixed(
-        2
-      )}s` + ` (${operationsPerSecond.toFixed(2)} ops/s, ${secondsPerOperation.toFixed(3)} s/op)`
+      `Completed ${totalOperations.toLocaleString(undefined, { maximumFractionDigits: 0 })} ` +
```

Comment:
```suggestion
      `Completed ${totalOperations} ` +
```
Why not just this?

Reply:
`toLocaleString()` is used to insert comma-separators like `123,456`

Reply:
I see, fine then.